utils/atomic-storybook/package.json

```diff
@@ -3,25 +3,38 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@storybook/addon-a11y": "6.3.12",
-    "@storybook/addon-controls": "6.3.12",
-    "@storybook/addon-docs": "6.3.12",
-    "@storybook/addon-essentials": "6.3.12",
-    "@storybook/addon-viewport": "6.3.12",
-    "@storybook/api": "6.3.12",
-    "@storybook/builder-webpack5": "6.3.12",
-    "@storybook/components": "6.3.12",
-    "@storybook/html": "6.3.12",
-    "@storybook/manager-webpack5": "6.3.12",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
     "@monaco-editor/react": "4.3.1",
+    "@storybook/addon-a11y": "7.0.7",
+    "@storybook/addon-controls": "7.0.7",
+    "@storybook/addon-essentials": "7.0.7",
+    "@storybook/addon-interactions": "^7.0.7",
+    "@storybook/addon-links": "^7.0.7",
+    "@storybook/addon-mdx-gfm": "^7.0.7",
+    "@storybook/addon-viewport": "7.0.7",
+    "@storybook/api": "7.0.7",
+    "@storybook/blocks": "^7.0.7",
+    "@storybook/components": "^7.0.7",
+    "@storybook/html": "7.0.7",
+    "@storybook/html-webpack5": "^7.0.7",
+    "@storybook/react": "^7.0.7",
+    "@storybook/react-webpack5": "^7.0.7",
+    "@storybook/testing-library": "^0.0.14-next.2",
+    "lit-analyzer": "^2.0.0-pre.2",
+    "lodash": "4.17.21",
     "monaco-editor-webpack-plugin": "6.0.0",
-    "lit-analyzer": "1.2.1",
+    "prop-types": "^15.8.1",
     "react": "16.14.0",
     "react-dom": "16.14.0",
```

Comment:
Can you check if we can 'bump' react up so that it's synced up/deduped with our other project.
@btaillon migrated storybook out due to this, so if we manage to sync them up, we could move storybook back into the fold and reduce the complexity